CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # UNRELEASED
 
+-   Update matrix-rust-sdk to `75683d268fc2e`
+
+**BREAKING CHANGES**
+
+-   Remove `SignedCurve25519` variant of `DeviceKeyAlgorithm`.
+
+-   One-time keys and fallback keys are now represented by `OneTimeKeyAlgorithm`
+    instead of `DeviceKeyAlgorithm`.
```

Comment:
`OneTimeKeyAlgorithm` is not exposed by these bindings, is it?

Reply:
Hmm, no, but it still seems worth talking about. What do you think?

Reply:
Removed in c159ace